lens/util/repo.go

```diff
@@ -201,10 +192,10 @@ func MakeGetActorCodeFunc(ctx context.Context, store adt.Store, next, current *t
 		// TODO accept a context, don't take the function context.
 		_, innerSpan := otel.Tracer("").Start(ctx, "GetActorCode")
 		defer innerSpan.End()
-		// Shortcut lookup before resolving
-		c, ok := actorCodes[a]
-		if ok {
-			return c, true
+
+		act, err := nextStateTree.GetActor(a)
+		if act != nil {
```

Comment:
```suggestion
		if err != nil {
```

Reply:
If we don't get an object, the code should continue with fallback logic.